### ISSUE TITLE
Female ETA ?

### ISSUE THREAD

**Issue by Gregorsaur** [boermansjo/ESX_HUD#16]
Hi mate,

I know you'be said you'd do females soon but, do you have any ETA ? 

Cheers

**Comment by XvenDeR** [comment]
I don't think he have much time doing this right now.

**Comment by boermansjo** [comment]
I see that there is still a lot of interest in this. To all, let me fix a date : 31 October.
I will force myself and give you the script you all want.

**Comment by Gregorsaur** [comment]
@boermansjo Legend :)

**Comment by amerem3** [comment]
Your a total legend!

**Comment by boermansjo** [comment]
> Hi mate,
> 
> I know you'be said you'd do females soon but, do you have any ETA ?
> 
> Cheers

Just to let you know, I have done some work offline. Some changes are not final. So I didn't updated this project. Sorry for that :/

**Comment by NewEdenJimW** [comment]
I am currently working on a onesync server and will definitely be using this, i love you man! This is going to be sick!

**Comment by boermansjo** [comment]
Hi, Sorry for the delay. Been sick last days... I will update the female part soon. Finding good outfits is a huge task. So there will be fewer female outfits.

**Comment by Gregorsaur** [comment]
That's fine mate, people can always add their own :D

**Comment by XvenDeR** [comment]
Yeah, no problem. When you update it with template we gonna add more, at least I will, when it will be clear how to do it. Or you always can leave a little tutorial how you do this, so we could help this process out.

**Comment by NewEdenJimW** [comment]
@boermansjo Just let us know if you need help, i am sure the community is more then willing to help and i cant wait to start using this and help with modifying this! <3

**Comment by landon678** [comment]
> 
> 
> I see that there is still a lot of interest in this. To all, let me fix a date : 31 October.
> I will force myself and give you the script you all want.

Awesome! I am super excited. I wish I could help with it, but I just started to learn about Lua, and all I know how to do is web code. If there is any leg work or information gathering I can do, let me know.

**Comment by CrazeDE** [comment]
Looking forward to this release!

**Comment by Gregorsaur** [comment]
Was this ever done?

**Comment by XvenDeR** [comment]
> Was this ever done?

no

**Comment by boermansjo** [comment]
No ETA for female yet... Sorry. I will do my best to have as soon as possible

**Comment by TristGit** [comment]
> No ETA for female yet... Sorry. I will do my best to have as soon as possible

Still looking forward to this, dont abandon this my friend.
